applications/audio/broadcast_config_tool/src/main.c

```diff
@@ -422,7 +453,7 @@ static void broadcast_config_print(const struct shell *shell,
 		int language = bt_audio_codec_cfg_meta_get_stream_lang(codec_cfg);
 
 		if (language > 0) {
-			char lang[4] = {"\0"};
+			char lang[4] = {'\0'};
```

Comment:
Maybe we could have a define for the length of language context length for handling this in generic way.

Reply:
I could add a define in the header file. In Zephyr they just do:
```C	
if (ret != 3) { /* Stream language is 3 octets */
	return -EBADMSG;
}
```
for instance, so there is no global define for this it seems